sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventFacadeEjb.java

```diff
@@ -355,6 +356,12 @@ public long count(EventCriteria eventCriteria) {
 					order.add(sortProperty.ascending ? cb.asc(expression) : cb.desc(expression));
 					expression = location.get(Location.ADDITIONAL_INFORMATION);
 					break;
+				case EventIndexDto.REPORTING_USER:
```

Comment:
Please also sort by last name, similar to how it's done in `ContactFacadeEjb:getContactFollowUpList` (currently lines 877ff on the dev branch) for both reporting user and responsible user